rust/src/ldap/ldap.rs

```diff
@@ -32,6 +33,13 @@ static mut LDAP_MAX_TX: usize = 256;
 
 static mut ALPROTO_LDAP: AppProto = ALPROTO_UNKNOWN;
 
+// app-layer-frame-documentation tag start: FrameType enum
+#[derive(AppLayerFrameType)]
+pub enum LdapFrameType {
+    Pdu,
```

Comment:
Could we have header and data ? Or is it not relevant for LDAP ?

Reply:
LDAP doesn't have a header per se, but just a message ID.

Reply:
Where are the operation and pdu length encoded ?

Reply:
They are encoded following ASN.1 syntax

Reply:
Like BER or DER encoding.
They have at least some tag stuff that we could define as the header...

But ok just to have `Pdu` for first version, thanks :-)